talk/setup.tex

```diff
@@ -31,6 +31,9 @@
   \end{beamercolorbox}%
 }
 
+\setbeamercolor{goodpractice head}{fg=white,bg=orange}
```

Comment:
```suggestion
\setbeamercolor{goodpractice head}{fg=white,bg=orange!50!black}
```
Or something of this kind, that is more dark so that the white writing is easier to read

Reply:
`orange!50!black`:
<img width="620" alt="image" src="https://user-images.githubusercontent.com/16205615/188658787-7b36e7e4-87be-4ca8-a392-07d4b7bf301e.png">

`orange!90!black`:
<img width="627" alt="image" src="https://user-images.githubusercontent.com/16205615/188658560-36f6e5af-4385-44ff-a350-b2f5a867d44e.png">


Current head:
<img width="619" alt="image" src="https://user-images.githubusercontent.com/16205615/188658942-0cd71fb1-e421-4605-b613-f0e4f396b32b.png">

Opinions?

Reply:
some 70-80 ? so that it stays orange (and not brown) but slightly darker. Actually the 90 is not bad already

Reply:
85 now. I will merge, and we can tweak it once we see the slides on a projector.